src/components/Common/LogoGrid.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Row, Col } from '../grid'
+import styled from 'styled-components'
+import Image from '../Common/Image'
+import ExternalAnchor from '../Common/ExternalAnchor'
+
+const Column = styled(Col)`
+  max-height: 108px;
+  display: flex;
+  align-items: center;
+  padding-bottom: ${props => props.theme.spacing[1]};
+`
+
+const LogoGrid = ({ logos = [] }) =>
+  logos ? (
+    <Row>
+      {logos.map(logo => (
+        <Column
+          width={[1 / 2, 1 / 2, 1 / 2, 1 / 2, 1 / 3, 1 / 4]}
+          key={logo.id}
+        >
+          {logo.url ? (
+            <ExternalAnchor href={logo.url}>
+              <Image
+                image={logo.image}
+                width="250px"
+                style={{ filter: 'grayscale(1)', saturate: '0' }}
+              />
+            </ExternalAnchor>
+          ) : (
+            <Image
+              image={logo}
```

Comment:
Is it not supposed to also be `image={logo.image}` here?

Reply:
Unfortunately not...but I agree that this looks weird :thinking: The discrepency is because the way logos are added to a section is different depending on the page

It's because, if there is a url associated with the logo (e.g. on the [About Us page](https://github.com/yldio/yld.io/blob/master/src/pages/about-us.js#L161-L173)), then the shape of the object from Contentful comes from this graphql query:
```
logos {
            name
            url
            image {
              title
              file {
                url
              }
              fluid(maxWidth: 250) {
                ...GatsbyContentfulFluid_withWebp
              }
            }
          }
```

Otherwise, as in the [speciality template](https://github.com/yldio/yld.io/blob/master/src/templates/speciality.js#L164-L171), the `logo` = `image` only. I.e. this is the shape:
```
logos {
        id
        title
        file {
          url
          fileName
        }
      }
```

Unsure what the best way is to make this cleaner... :thinking:

Reply:
Wow. As misleading as it is refactoring is not a priority - we have other things to focus on but thanks for the explanation. It is for sure confusing for people who don't know.

One thing we should do because it should not mean too much extra work is document these shapes in `Readme`s for future developers that join the project.

Reply:
We chatted about this via Slack, but https://github.com/yldio/yld.io/pull/190/commits/a6ccb1ece84e3ba27a52954aaa51f67f54b32a9d seems to provide sufficient clarity for now.

Going to add the readme addon to storybook under another ticket though. Good to have either way.